tasks/gulp/compile-assets.js

```diff
@@ -25,6 +25,9 @@ const postcsspseudoclasses = require('postcss-pseudo-classes')({
 // Compile CSS and JS task --------------
 // --------------------------------------
 
+// check if destination flag is public (this is the default)
+const isPublic = taskArguments.destination === 'public' || false
```

Comment:
It's also [used to build](https://github.com/alphagov/govuk-frontend/blob/main/package.json#L17) the `package` folder. I think technically this task isn't very "locked down" so you can actually run `gulp scss:compile --destination [any folder]` and it will compile into that folder, so I thought it best not to assume anything... Probably something we need to tackle when we look at all the gulp tasks.